test/unit/subnet/ipv6_test.rb

```diff
@@ -0,0 +1,149 @@
+require 'test_helper'
+
+class SubnetIpv6Test < ActiveSupport::TestCase
+  def setup
+    User.current = User.find_by_login "admin"
+  end
+
+  test "should have a network" do
```

Comment:
Use shoulda-matchers, `should validate_presence_of(:network)` does the same w/o looking at the db